src/transformers/models/dinov2/modeling_dinov2.py

```diff
@@ -858,8 +892,9 @@ def forward(
                 output = (feature_maps,) + outputs[2:]
             return output
```

Comment:
if not return dict should also return the `cls_tokens`. Unless in order not to be breaking you have to set `return_dict` to `True` to get the `cls_tokens`?

Reply:
Weird this wasn't catched by the `test_model_outputs_equivalence` test.. thanks for spotting!

Reply:
Fixed